entries/p/pro.btcturk.com.json

```diff
@@ -0,0 +1,16 @@
+{
+  "BtcTurk Pro": {
+    "domain": "pro.btcturk.com",
+    "tfa": [
+      "sms",
+      "email",

```

Comment:
https://www.btcturk.com/yardim/guvenlik Only mentions SMS and TOTP. Are you sure email authentication is possible?

Reply:
Yes is possible, if your account is not enable SMS or TOTP, email authentication necessarily activating.